scripts/qiita

```diff
@@ -279,6 +280,25 @@ def load_parameters(fp, table, name):
 
 
 @db.command()
+@click.option('--fp', required=True, type=click.Path(resolve_path=True,
+              readable=True, exists=True), multiple=True,
+              help='Path to the raw data file. This option can be used '
+              'multiple times if there are multiple raw data files.')
+@click.option('--fp_type', required=True, multiple=True, help='Describes the '
+              'contents of the file. Pass one fp_type per fp.',
+              type=click.Choice(get_filepath_types().keys()))
+@click.option('--study', required=True, type=int,
```

Comment:
Is it safe to assume the type is going to be > 0? if that's the case range options may be useful http://click.pocoo.org/5/options/#range-options